.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
```

Comment:
I've been meaning to delete this file elsewhere. Let's not add it back here. Customers rarely use it correctly and we end up having to delete it more than anything.